lib/components/Text/Text.tsx

```diff
@@ -53,9 +49,9 @@ export const Text = ({
     <TextContext.Provider value={textContextValue}>
       <Box
         id={id}
-        display={!isListItem ? 'block' : undefined}
+        display="block"
         component={component}
-        className={classnames(textStyles, isListItem && styles.listItem)}
+        className={classnames(textStyles)}
```

Comment:
```suggestion
        className={textStyles}
```

Reply:
I'll do this manually and drop the import as well.